src/mem.rs

```diff
@@ -198,46 +198,56 @@ impl Index<usize> for MmapMemory {
 /// Wrapper aroud a memory allocation that is aligned to x86 HugePages
 /// (`0x20_0000`). Intended for testing purposes only
 #[cfg(test)]
-#[expect(
+#[allow(
```

Comment:
This raised a warning when running `cargo test`, so I changed it back to `allow`.

cc @n0toose

Reply:
+1